docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -6,7 +6,7 @@ import { alpha } from '@mui/material/styles';
 export default function AppFrameBanner() {
   return FEATURE_TOGGLE.enable_docsnav_banner ? (
     <Link
-      href="https://next.mui.com/x/whats-new/"
+      href="https://mui.com/x/whats-new/"
```

Comment:
I think that we can replace this at the same time

```suggestion
      href="https://mui.com/blog/mui-x-v6/"
```

Reply:
The blog announcement is coming right after the release, so I'd recommend that we still use the banner pointing to the "whats-new" page.

We also plan on keeping the "what's new" page for a while, even after the blog post is published. 
I think it has the potential to continuously get eyes on the new version. The blog post notification fades away over time.

Reply:
> We also plan on keeping the "what's new" page for a while, even after the blog post is published.
I think it has the potential to continuously get eyes on the new features. The blog post notification fades away over time.

@joserodolfofreitas The link in the side nav feels a bit out of context: if you start a new project and you are on v6 , you don't really care, if you are on v5 and you are not supposed to read this doc, you don't really care. But no real harm so why not.

I would recommend, that we eventually: 

1. Add a link to the blog post in the migration pages https://mui.com/x/migration/migration-data-grid-v5/. I think that it's important that we give the developer a reason why they should upgrade as a counterweight to why not (the BCs).
2. Update https://v5.mui.com/x/introduction/ so the what's new link points to the blog post once live
3. Delete https://mui.com/x/whats-new/ as a HTML page and have the "what's new" link point to the blog post https://deploy-preview-36398--material-ui.netlify.app/blog/mui-x-v6/. I mean, the blog post looks like a better version of it, I don't see why we should have both contents live to duplicate.